images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+# if there is a deploymentconfig for the given service
+if [[ "oc -n ${OPENSHIFT_PROJECT} get deploymentconfigs -l service=${SERVICE_NAME}" ]]; then
```

Comment:
@rtprio 
this looks like a check for a string for me which will always be true?

Reply:
corrected.